go.mod

```diff
@@ -5,7 +5,7 @@ go 1.20
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.7
 	github.com/Masterminds/semver/v3 v3.2.1
-	github.com/Microsoft/go-winio v0.6.2
+	github.com/Microsoft/go-winio v0.6.1
```

Comment:
An important thing to note is that this update does not seem to contain bug fixes or security fixes that we need.   The revert would be trickier otherwise.